build.gradle

```diff
@@ -178,6 +178,12 @@ dependencyManagement {
     dependencySet(group: 'com.google.guava', version: '28.2-jre') {
       entry 'guava'
     }
+    dependencySet(group: 'org.apache.tomcat.embed', version: '9.0.30') {
+      entry 'tomcat-embed-core'
+    }
+    dependencySet(group: 'org.apache.tomcat.embed', version: '9.0.30') {
+      entry 'tomcat-embed-websocket'
+    }
```

Comment:
Can be squashed under same `dependencySet` as group and version match

Reply:
done